content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,387 @@
+      text: 'Why should I install the Datadog Agent on my cloud instances?'
+    - link: 'https://www.datadoghq.com/blog/gke-dashboards-integration-improvements/'
+      tag: 'Blog'
+      text: 'New GKE dashboards and metrics provide deeper visibility into your environment'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-private-service-connect/'
+      tag: 'Blog'
+      text: 'Access Datadog privately and monitor your Google Cloud Private Service Connect usage'
+    - link: 'https://www.datadoghq.com/blog/track-bigquery-costs-performance/'
+      tag: 'Blog'
+      text: 'Monitor BigQuery with Datadog'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-cost-management/'
+      tag: 'Blog'
+      text: 'Empower engineers to take ownership of Google Cloud costs with Datadog'
+    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+      text: 'Collect traces, logs, and custom metrics from your Google Cloud Run services with Datadog'
+      tag: 'Blog'
+---
+
+## Overview
+
+Integrating your Google Cloud projects with Datadog allows you to monitor metrics from your GCE and GKE compute instances within the Datadog platform.
+This guide provides an overview of how to quickly get started with the Datadog Google Cloud integration.
+
+## Prerequisites
+1) Create a [Datadog account][1]
+2) Set up a [Service Account][2] in Google Cloud
+3) Review these Google Cloud Prerequisites:
+
+- If your organization restricts identities by domain, you must add Datadog's customer identity `C0147pk0i` as an allowed value in your policy.
+- Ensure that any projects being monitored are **not** configured as [scoping projects][42] that pull in metrics from multiple other projects.
+
+- The Google Cloud integration requires the below APIs to be enabled **for each of the projects** you want to monitor:
+
+[Cloud Monitoring API][7] 
+: Allows Datadog to query your Google Cloud metric data.
+
+[Compute Engine API][8] 
+: Allows Datadog to discover compute instance data.
+
+[Cloud Asset API][9]
+: Allows Datadog to request Google Cloud resources and link them to relevant metric tags.
+
+[Cloud Resource Manager API][10] 
+: Allows Datadog to append metrics with the correct resources and tags.
+
+[IAM API][11]
+: Allows Datadog to authenticate with Google Cloud.
+
+[Google Cloud Billing API][12] 
+: Enables accurate billing for API calls.
+
+<div class="alert alert-info">You can confirm if these APIs are enabled by heading to <a href="https://console.cloud.google.com/apis/dashboard"><b>Enabled APIs & Services</a></b> under each project in your Google Cloud workspace.</div>
+
+
+## Setup 
+
+Integrate Datadog with [Google Cloud][4] using [service account impersonation][5] and project discovery. Service account impersonation enables Datadog to automatically discover and monitor new projects as they are created within the assigned scope, providing seamless monitoring as your Google Cloud environment grows.
```

Comment:
This is a little misleading service account impersonation doesn't allow for the discovery would likely rephrase like so: 
```suggestion
Integrate Datadog with [Google Cloud][4] using [service account impersonation][5] and project discovery. This enables Datadog to automatically discover and monitor new projects as they are created within the assigned scope, providing seamless monitoring as your Google Cloud environment grows.
```